src/backend/catalog/ag_label.c

```diff
@@ -177,8 +177,13 @@ int32 get_label_id_from_entity(agtype_value *entity, const char *graph_name)
                                                    AG_DEFAULT_LABEL_EDGE;
     }
 
-    gcd = search_graph_name_cache(graph_name);
-    lcd = search_label_name_graph_cache(label_name, gcd->oid);
+    if (graph_oid == 0)
```

Comment:
What if `0` is a valid `graph_oid`?

Reply:
As per my understanding, the `OID` cannot be 0. However, I can modify the function and define `graph_oid` as a pointer, by doing so, I can pass `NULL` to represent an invalid `graph_oid`.